### DISCUSSION TITLE
Issue with Ownable Constructor in Stablecoin Project

### DISCUSSION THREAD

**Discussion by pherm0007** [Cyfrin/foundry-full-course-cu#2811] [General]
Just wanted to flag an issue I ran into while working on the decentralized stablecoin project. It looks like the latest version of OpenZeppelin’s Ownable (v5.1.0) has made a change where the Ownable constructor now expects an initialOwner parameter, instead of just setting msg.sender as the owner by default like it used to.

So when I tried using Ownable  without any argument, it threw an error. It took me a bit to realize that the constructor now needs an address passed in to set up ownership properly. Not sure if others following the course might get tripped up by this too, so I thought I'd let you know

Appreciate you checking this out

**Comment by EngrPips** [comment]
This is addressed in the `chronological-update` section of this GitHub repo.

**Reply by pherm0007** [reply]
Thanks I will check it out now

**Reply by EngrPips** [reply]
Yeah, do.
